src/index.html

```diff
@@ -22,6 +22,308 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <nav class="nav">
+        <div class="nav__img">
+          <img
+            src="./images/logo.png"
+            alt="Logo"
+          />
+        </div>
+        <div class="nav__container">
+          <div class="nav__items">APPLE</div>
+          <div class="nav__items">SAMSUNG</div>
+          <div class="nav__items">SMARTPHONES</div>
+          <div data-qa="nav-hover" class="nav__items">LAPTOPS & COMPUTERS</div>
+          <div class="nav__items">GADGETS</div>
+          <div class="nav__items">PHOTO</div>
+          <div class="nav__items">VIDEO</div>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__container">
+        <div>
```

Comment:
["use main tag for cards container"](https://github.com/mate-academy/layout_catalog?tab=readme-ov-file#frontend-practice-with-catalog-page)